apps/omg_watcher/priv/swagger/watcher_api_specs.yaml

```diff
@@ -0,0 +1,588 @@
+      operationId: status
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      responses:
+        200:
+          $ref: '#/responses/StatusOpId'
+        500:
+          $ref: '../../../omg_rpc/priv/swagger/shared.yaml#/InternalServerError'
+  /utxo.get_utxos:
+    post:
+      tags:
+        - UTXO
+      summary: Gets all utxos belonging to the given address.
+      description: >
+        Note that this is a performance intensive call and should only be used if the chain is byzantine and the user needs to retrieve utxo information to be able to exit.
+        Normally an application should use the Informational API's [Account - Get Utxos](http://TODO) instead.
+        This version is provided in case the Informational API is not available.
+      operationId: getUtxos
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      requestBody:
+        $ref: 'requests.yaml#/AddressBodySchema'
+      responses:
+        200:
+          $ref: '#/responses/GetUtxosOpId'
+        500:
+          $ref: '../../../omg_rpc/priv/swagger/shared.yaml#/InternalServerError'
+  /utxo.get_challenge_data:
+    post:
+      tags:
+        - UTXO
+      summary: Gets challenge data for a given utxo exit.
+      description:
+      operationId: getUtxoChallenge
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      requestBody:
+        $ref: 'requests.yaml#/UtxoPosBodySchema'
+      responses:
+        200:
+          $ref: '#/responses/GetUtxoChallengeOpId'
+        500:
+          $ref: '../../../omg_rpc/priv/swagger/shared.yaml#/InternalServerError'
+  /utxo.get_exit_data:
+    post:
+      tags:
+        - UTXO
+      summary: Gets exit data for a given utxo.
+      description:
+      operationId: getUtxoExit
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      requestBody:
+        $ref: 'requests.yaml#/UtxoPosBodySchema'
+      responses:
+        200:
+          $ref: '#/responses/GetUtxoExitOpId'
+        500:
+          $ref: '../../../omg_rpc/priv/swagger/shared.yaml#/InternalServerError'
+  /transaction.submit:
+    post:
+      tags:
+        - Transaction
+      summary: Sends transaction to Child chain.
+      description: Watcher passes signed transaction to the child chain only if it's secure (better explaination needed)
+      operationId: submit
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      requestBody:
+        $ref: 'requests.yaml#/TransactionBodySchema'
+      responses:
+        200:
+          $ref: '#/responses/SubmitOpId'
+        500:
+          $ref: '../../../omg_rpc/priv/swagger/shared.yaml#/InternalServerError'
+  /inflight_exit.get_data:
+    post:
+      tags:
+        - InflightExit
+      summary: Gets exit data for an in-flight exit. 
+      description: Exit data are arguments to `startInFlightExit` root chain contract function.
```

Comment:
Summary and description are soo similar, what do you think about removing description?

Reply:
I'll keep it as might be useful for ppl on testnet